package.json

```diff
@@ -18,5 +19,17 @@
   "bugs": {
     "url": "https://github.com/thsvr/Microverse-606-testing-jest/issues"
   },
-  "homepage": "https://github.com/thsvr/Microverse-606-testing-jest#readme"
+  "jest": {
+    "runner": "groups",
+    "verbose": true
+  },
+  "homepage": "https://github.com/thsvr/Microverse-606-testing-jest#readme",
+  "devDependencies": {
+    "@babel/preset-env": "^7.8.6",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.20.1",
+    "jest": "^25.1.0",
+    "jest-runner-groups": "^1.1.1"
+  }
 }
```

Comment:
Use the `contributors` field instead of `author` for more than one person: https://docs.npmjs.com/files/package.json#people-fields-author-contributors.

Reply:
We had no idea about that, thanks!